src/crypto/aes256ctr.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2022 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
```

Comment:
We have a `src/common` directory already. Would that be a good place to put this crypto stuff?